pyBumpHunter/bumphunter_1dim.py

```diff
@@ -327,6 +332,10 @@ def _scan_hist(self, hist, ref, w_ar, ih: int):
 
             min_width :
                 Width of the window corresponding to the minimum p-value (integer).
+
+            norm_scale :
```

Comment:
I am not sure to understand what you mean here.
The objective of side-band normalization is to compute a noralization scale factor.
The `norm_scale` variable is just here to store the computed scale so it can be retrieved.